api/agent/drivers/docker/docker.go

```diff
@@ -101,13 +148,36 @@ func NewDocker(conf drivers.Config) *DockerDriver {
 		}
 	}
 
+	imagesBeforeLoad, err := driver.docker.ListImages(context.Background())
+
 	if conf.DockerLoadFile != "" {
 		err = loadDockerImages(driver, conf.DockerLoadFile)
```

Comment:
I think there's an output stream in args:

https://godoc.org/github.com/fsouza/go-dockerclient#LoadImageOptions

if this works, it's better than hitting the list images twice.

Reply:
We have to parse raw logs if we use that. Is that something we are interested in?

Reply:
yeah, it doesn't look very structured. :-(